content/registries/_index.md

```diff
@@ -13,3 +13,61 @@ weight: 23
 oc patch image.config.openshift.io/cluster -p \
 '{"spec":{"allowedRegistriesForImport":[{"domainName":"my.own.registry.example.com:8888","insecure":true}],"registrySources":{"insecureRegistries":["my.own.registry.example.com:8888"]}}}' --type='merge'
 ```
+
+# Configure custom/insecure registry to search path in OCP 4.x
+
+At this moment, there seems to be no way to remove/modify or append any additional entries to unqualified-search-registries line in /etc/containers/registries.conf file as of now (OCP <=4.3).
```

Comment:
What about this?:

In OpenShift 4, the registries configuration is managed by the [Image Registry Operator](https://docs.openshift.com/container-platform/4.3/registry/configuring-registry-operator.html). In order to modify registries parameter, it is only required to modify the `image.config.openshift.io/cluster` object, that manages  the `/etc/containers/{policy.json,registries.conf}` files content under the hood.

In OpenShift versions prior to 4.4 a missing feature of the operator is the ability to remove, modify or append any additional entry to the `unqualified-search-registries` line in the `/etc/containers/registries.conf` file to allow search in insecure registries.